src/main.js

```diff
@@ -1,9 +1,11 @@
-require("slint-ui");
-let ui = require("../ui/appwindow.slint");
+
+import * as slint from "slint-ui";
+
+let ui = slint.loadFile("ui/appwindow.slint");
 let window = new ui.AppWindow();
 
-window.request_increase_value.setHandler(() =>
-    window.counter = window.counter + 1
-);
+window.request_increase_value = function () {
+    window.counter = window.counter + 1;
+};
 
 window.run();
```

Comment:
It works without, but this way it's more evident that run() returns a promise, instead of calling run() and then letting the future be resolved in the background.

```suggestion
await window.run();
```